### PR TITLE
apply new grc20 version

### PR DESCRIPTION
**Pull request by notJoon**
- Modify previous code to match the modified type since GRC20 has recently changed.
- Added airdrop `Refund` function.

### CODE REVIEW

File: airdrop/airdrop.gno [hunk 1]
```diff
@@ -12,8 +13,13 @@ var (
 	ErrInvalidClaimsCount  = errors.New("INVALID_CLAIMS_COUNT")
 	ErrFirstClaimIsNot64   = errors.New("FIRST_CLAIM_MUST_BE_MULT_64")
 	ErrInsufficientBalance = errors.New("INSUFFICIENT_BALANCE")
+	ErrRefundNotAllowed    = errors.New("refund not allowed")
+	ErrTooEarly            = errors.New("too early")
+	ErrNoBalance           = errors.New("no balance to refund")
 )
 
+var timeNow = time.Now
```
Comment (r3v4s) [inline airdrop/airdrop.gno:21]: typo??
1. time.Now() 
2. time.Now().Unix()

**Review by r3v4s — CHANGES_REQUESTED** [review]
one typo I think

File: airdrop/airdrop.gno (hunk 1 above)
Comment (notJoon) [reply airdrop/airdrop.gno:21]: it's valid syntax

Comment (notJoon) [reply airdrop/airdrop.gno:21]: it's valid syntax. this variable can be use like function later.

```go
var timeNow = time.Now

func main() {
	println(timeNow().Unix())
}
```

**Review by r3v4s — APPROVED** [review]

File: airdrop/airdrop.gno (hunk 1 above)
Comment (r3v4s) [reply airdrop/airdrop.gno:21]: what an weird syntax :(

**Pull request merged**